crates/api/src/op_store.rs

```diff
@@ -69,6 +70,7 @@ impl PartialOrd for StoredOp {
 }
 
 /// The API that a kitsune2 host must implement to provide data persistence for kitsune2.
+#[automock]
```

Comment:
Auto-mocking this trait. I'll apply this pattern to the other fetch test files as well.